modules/acquia_cms_site_studio/modules/acquia_cms_site_studio_cm/README.md

```diff
@@ -0,0 +1,35 @@
+# About
+
+Contains configuration for config ignore & config split specific to site studio module, that allow better configuration
+management between core config and site studio.
+
+## Usages
+In order to utilise this module's functionality, user needs to add scaffolding to their project.
+they can run command `composer config --json extra.drupal-scaffold.allowed-packages ["acquia/acquia_cms_site_studio"] --merge && composer instal`
```

Comment:
```suggestion
they can run command 
```composer config --json extra.drupal-scaffold.allowed-packages ["acquia/acquia_cms_site_studio"] --merge && composer install```
```
The install

Reply:
Done